scripts/test-manual-e2e.sh

```diff
@@ -76,11 +76,14 @@ npm pack
 PACKAGE=$(pwd)/react-native-$PACKAGE_VERSION.tgz
 success "Package bundled ($PACKAGE)"
 
+node set-rn-template-version.js "file://$PACKAGE"
+success "React Native version changed in the template"
+
 project_name="RNTestProject"
 
 cd /tmp/
 rm -rf "$project_name"
-react-native init "$project_name" --version $PACKAGE
+npx @react-native-community/cli init "$project_name" --template "$repo_root"
```

Comment:
This points to the `latest` which currently is 1.x and 0.60+ is going to be working with 2.x. Maybe it's worth to pass the version/tag from the arguments?

Reply:
We can just call $repo_root/cli.js? This will take whatever React Native has in source.

Reply:
Calling `cli.js` would be the best solution. I'll change that.